tests/e2e/files/playbooks/post_processing.yml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
The name of this file is confusing. This playbook is for a specific testcase, is not for general purposes as "print_event". The name should include what test belongs to.